web3/_utils/caching.py

```diff
@@ -39,7 +39,7 @@ def generate_cache_key(value: Any) -> str:
             in value
         )))
     else:
-        raise TypeError("Cannot generate cache key for value {0} of type {1}".format(
-            value,
-            type(value),
-        ))
+        raise TypeError(
```

Comment:
Actually, my editor automatically reformats the line because I set the max length of the line to be 70, I forgot that in this project it is set to 100 chars.